pkg/stats/stream.go

```diff
@@ -0,0 +1,174 @@
+package stats
```

Comment:
Don't think we are using this.

Reply:
All these "unneed" code above, remove them will cause so much code change, because the code is cross reference in used code and unused code. In my plan, the first step is copy original code from sfu we used ( avoid build error). Next step, we can refactor the sfu code from ion-sfu, remove the unused struct & field & import library.If we use one pr to do this, the code is hard to review, because all code is new in commit, you can't see which code is modified because remove the unused code.So I will add a new task to clean code in sfu. @boks1971 @davidzhao  Do you think it is make sense?

Reply:
I have tried to copy used files only but not modify the code, but a lot build error is happend...

Reply:
Totally makes sense @cnderrauber . One step at a time :-) 

The only thing I wanted to check was if it is possible to stick all these files under `pkg/sfu` in `livekit-server` and clean up that directory? Or is that also painful. I think it is fine to have it at this level, but for some reason, my mind wants to keep that separate for now so that it is easy to see what all came from `ion-sfu` :-) and then we can slowly assimilate it.

Reply:
Do you means move twcc/buffer/other folders to sfu? I think that's ok, should modify import lines in  the code which import these packages bouth in sfu and livekit-server.  I can add another commit to this pr to do this, so you can just review the second commit's file change.

Reply:
Yeah, that's what I meant. Just keep all the `ion-sfu` stuff in one directory for now. Thank you!